utils/Wordpress/getCategories.ts

```
@@ -0,0 +1,16 @@
+import { IWPCategory } from "@/types/category";
+import { IWordpressPost } from "@/types/post";
+
+export function getCategoryDetailsFromPost(post: IWordpressPost): IWPCategory[] {
+  const categories: IWPCategory[] = [];
+  if (post._embedded && post._embedded["wp:term"]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)